src/main/java/io/confluent/connect/elasticsearch/DataConverter.java

```diff
@@ -154,7 +160,7 @@ static Schema preProcessSchema(Schema schema) {
     }
   }
 
-  private static SchemaBuilder copySchemaBasics(Schema source, SchemaBuilder target) {
+  SchemaBuilder copySchemaBasics(Schema source, SchemaBuilder target) {
```

Comment:
nit: Leave a comment as below that is `//visible for testing`?

Also, up to you if you want to make it `protected` instead. (this one and the other package private method used for testing below).